### PR TITLE
 Build only on PR

### PR DESCRIPTION
**Pull request by petra-sargent**
This is the feature documentation for Build only on PR.

### CODE REVIEW

**Comment by petra-sargent** [comment]
@krames  Please see the Feature Documentation for Build on PR.

File: _general/projects/build_trigger.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+| Push a Commit                	|                       Yes                      |	|			No			|
+| Push a Tag                  	|                       Yes                      |	|                      No                      	|
+| Open a Pull Request   	|                       No                       |	|                      Yes                     	|
+| Update a Pull Request 	|                       Yes                      |	|                      Yes                     	|
+| Merge a Pull Request  	|                       Yes                      |	|                      Yes                     	|
+
+
+## Configuring Branches
+
+You can run builds for all branch, exclude branches from triggering builds, and run builds only for certain branches.
```
Comment (krames) [inline _general/projects/build_trigger.md]: How about 

> You can run builds for all branches, only certain branches, or exclude branches.

File: _general/projects/build_trigger.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+ 
+### Excluding Branches From Triggering Builds
+
+To exclude branches from triggering a build:
+
+1. Go to Project Settings
+
+2. Select the _Build Triggers_ tab
+
+3. Select the _Don't run builds for these branches_ button
```
Comment (krames) [inline _general/projects/build_trigger.md]: Radio button?

File: _general/projects/build_trigger.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+4. Enter either full branch names like "master" or a regular expressions like "^release.*$".
```
Comment (krames) [inline _general/projects/build_trigger.md]: > Enter either full branch names like "master" for exact matches or a regular expressions like "^release.*$".

File: _general/projects/build_trigger.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+### Running Builds Only For These Branches
+
+To run builds only on certain branches: 
+
+1. Go to Project Settings
+
+2. Select the _Build Triggers_ tab
+
+3. Select the _Run build for these branches only_ button
```
Comment (krames) [inline _general/projects/build_trigger.md]: radio button?

File: _general/projects/build_trigger.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+You can trigger a build when committing, merging, and pushing tags to your project.  This is the default behavior. See [working with tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) for more information on pushing tags.   
+
+
+To trigger a build on commit, builds and tags:
+
+1. Go to Project Settings.
+
+2. Select the _Build Triggers_ tab.
+
+3. Select the _Build on commit, merge, and tags_ button.
```
[the remark this answers is not in the document]
Comment (petra-sargent) [reply _general/projects/build_trigger.md]: Yes! I did consult with Josh and the terminology is radio button. Thank you!

File: _general/projects/build_trigger.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+4. Click Save Configuration.
+
+
+### Triggering Builds Only On Pull Requests
+
+You can trigger a build when opening, merging, and updating a Pull Requests (PR).
```
[the remark this answers is not in the document]
Comment (petra-sargent) [reply _general/projects/build_trigger.md]: Yes!

File: _general/projects/build_trigger.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+
+To trigger a build only on PRs:
+
+1. Go to Project Settings.
+
+2. Select the _Build Triggers_ tab.
+
+3. Select the _Build only on PR events_ button.
```
[the remark this answers is not in the document]
Comment (petra-sargent) [reply _general/projects/build_trigger.md]: Yes

File: _general/projects/build_trigger.md (hunk 2 above)
Comment (petra-sargent) [reply _general/projects/build_trigger.md]: Yes!

File: _general/projects/build_trigger.md (hunk 3 above)
Comment (petra-sargent) [reply _general/projects/build_trigger.md]: Do we need to add more information about the regular expression?  Something like..."Enter either full branch names like "master" for exact matches or a regular expression like "^release.*$" for a search pattern."

File: _general/projects/build_trigger.md (hunk 4 above)
Comment (petra-sargent) [reply _general/projects/build_trigger.md]: Yes!

File: _general/projects/skipping-builds.md [hunk 8]
```diff
@@ -13,7 +13,7 @@ tags:
   - builds
   - testing
   - skipping
-  - skip
+  - skip:
```
Comment (joesiewert) [inline _general/projects/skipping-builds.md]: The `:` here is causing the build to fail

File: _general/projects/build_trigger.md (hunk 1 above)
Comment (petra-sargent) [reply _general/projects/build_trigger.md]: Sounds good!

File: _general/projects/skipping-builds.md (hunk 8 above)
Comment (petra-sargent) [reply _general/projects/skipping-builds.md]: Got it! Thanks!

**Pull request merged**